app/views/submissions/_annotation_global.html.erb

```diff
@@ -0,0 +1,4 @@
+<div class="base-global-annotation-form global-annotation-form">
+  <h1> Global annotation </h1>
```

Comment:
Nit: per accessibility best practices, only the main heading of a page should use an h1 tag; should use a lower-ranked heading element here

Reply:
I am using a `h1` tag because of the styling specific to `.annotationSummary h1`

<img width="310" alt="Screenshot 2022-11-27 at 14 52 41" src="https://user-images.githubusercontent.com/9074856/204156631-7889155b-51a1-4874-8133-f695ebf8e5ac.png">

Until recently, it was used for the annotations box 

![Screenshot 2022-11-27 at 14 54 07](https://user-images.githubusercontent.com/9074856/204156715-6f45d6fb-ecac-4043-8e90-30ddf84a3392.png)

In that case, should I update the style to use a `h2` tag or similar?

Reply:
Replaced with a `div`, and simplified the styles